### ISSUE TITLE
Getting error while connecting the bluetooth 

### ISSUE THREAD

**Issue by nero002** [KundalikSuryawanshi/Generic-Printer#1]
Android version 14
API 34

[
![Screenshot 2024-02-15 at 5 57 17 PM](https://github.com/KundalikSuryawanshi/Generic-Printer/assets/38027375/7f2f9433-247a-41ac-b0cc-83100f04591d)
](url)

**Comment by dev-gauravs** [comment]
yes! This issue stems from Bluetooth permissions on newer Android APIs. Addressing it involves implementing runtime user permissions and ensuring proper exception handling.

**Comment by nero002** [comment]
Let me raise a PR for this. I have tested it and it's working fine.

**Comment by dev-gauravs** [comment]
Okay.

**Comment by nero002** [comment]
Don't have the permission can you please allow me

<img width="1420" alt="Screenshot 2024-03-15 at 11 20 37 PM" src="https://github.com/KundalikSuryawanshi/Generic-Printer/assets/38027375/9b033930-0f27-4b93-90ba-bbe79b351b29">

**Comment by dev-gauravs** [comment]
now you have access to repo : )
can you just share your email id?

**Comment by nero002** [comment]
dhiraj02gupta@gmail.com
